frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Tab-Sidebar-ODC-4116.feature

```diff
@@ -0,0 +1,133 @@
+Feature: Monitoring tab on the Sidebar and Health Checks
+    As a user, I should be able to see Monitoring tab on the sidebar and add Health Checks
+
+
+Background: 
+    Given user is at Developer perspective
+
+
+@smoke
+Scenario: Monitoring tab on the Sidebar for Deployments
```

Comment:
The scenario should be something like `Opening Monitoring dashboard from deployment sidebar`

Reply:
Updated